Formula/dzr.rb

```diff
@@ -0,0 +1,24 @@
+class Dzr < Formula
+  desc "Command-line Deezer.com player"
+  homepage "https://github.com/yne/dzr"
+  url "https://github.com/yne/dzr/archive/refs/tags/230504.tar.gz"
+  sha256 "b7c50f1e75dd9cc75b56ede33ab4e9e1d09cbfce09bfaa2e9715f99896361447"
+  license "Unlicense"
+  head "https://github.com/yne/dzr.git", branch: "master"
+
+  depends_on "dialog"
+  depends_on "jq"
+  depends_on "mpv"
+  depends_on "openssl@1.1" # to match curl one
+  uses_from_macos "curl"
+
+  def install
+    bin.install "dzr", "dzr-url", "dzr-dec", "dzr-srt", "dzr-id3"
+  end
+
+  test do
+    ENV.remove "DZR_CBC"
```

Comment:
Sorry, my last suggestion was supposed to be:
```suggestion
    ENV.delete "DZR_CBC"
```
Please do squash your commits together. Thanks!

Reply:
Sure no problem, Done